CachetHQ/Cachet
# Custom meta descriptions per incident

Some additional features to close #2895. It's now possible to add / edit a custom meta title or description to an incident.

### What has been done:
- Created two fields when creating / editing an incident for the meta title and description
- Added a trait for updating the meta values of a Model since I discovered that the `Meta` model is Polymorphic, reused that in the command-handlers.
- The `UpdateIncidentCommand` now accepts metadata as a parameter
- Updated the frontend templates to show the custom meta title or description if set

### How to test:
- Make sure [your Cachet is up and running](https://docs.cachethq.io/docs/installing-cachet).
- Log into the dashboard and create an incident with a custom meta description and title
- Acknowledge your custom meta title an description on the incident details page
- Acknowledge the standard meta title an description on the incident details page when you haven't configured a custom one

### Notes:
- This entire PR is resting on the assumption that I can use the `Meta` model for saving this kind of data. Feel free to suggest otherwise.
